### PR TITLE
VxDesign: Add clerk signature image/caption custom content to ballots

### LINKED ISSUE #4481: NH local ballot template - clerk signature
https://votingworks.slack.com/archives/C060FEVB2L8/p1704400374864919

### PR DESCRIPTION
**Pull request by jonahkagan**
## Overview

Closes #4481 

Builds on #4495 to allow uploading a clerk signature image and caption for a precinct split. The image and caption are rendered in the ballot header.

## Demo Video or Screenshot

https://github.com/votingworks/vxsuite/assets/530106/fc39ba69-3b38-42e9-ba29-7a804da499f4

## Testing Plan
- Updated automated tests
- Manual test that layout works with signatures of varied length/image size (captured in video)

## Checklist

- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.

- [ ] I have added a screenshot and/or video to this PR to demo the change
- [ ] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: apps/design/frontend/src/geography_screen.tsx [hunk 1]
```diff
@@ -538,66 +545,116 @@ function PrecinctForm({
...
+                            alt="Clerk Signature"
+                            style={{ height: '3rem', marginBottom: '0.5rem' }}
+                          />
+                        )}
+                        <FileInputButton
+                          accept="image/svg+xml"
+                          onChange={(e) => {
+                            const file = e.target.files?.[0];
+                            if (!file) return;
+                            const reader = new FileReader();
```
Comment (eventualbuddha) [inline apps/design/frontend/src/geography_screen.tsx:607]: I suggest, before reading this file, that we pick a maximum file size. Perhaps 5MB? `File` has a `size` property in bytes that we can check.

**Review by eventualbuddha — APPROVED** [review]
Thanks for the video. How are we expecting people to create these SVGs? Or are we going to do it for them by just embedding an image that they provide?

**Comment by jonahkagan** [comment]
> How are we expecting people to create these SVGs? Or are we going to do it for them by just embedding an image that they provide?

I'm not sure, but I think it's a good question since this is a potential sticking point.

File: apps/design/frontend/src/geography_screen.tsx [hunk 2]
```diff
@@ -538,66 +542,56 @@ function PrecinctForm({
...
-                        value={split.name}
-                        onChange={(e) =>
-                          setPrecinct({
-                            ...precinct,
-                            splits: replaceAtIndex(precinct.splits, index, {
-                              ...split,
-                              name: e.target.value,
-                            }),
+              {precinct.splits.map((split, index) => {
+                return (
```
[the remark this answers is not in the document]
Comment (jonahkagan) [reply apps/design/frontend/src/geography_screen.tsx:549]: Thanks, can't remember why I changed that lol

File: apps/design/frontend/src/geography_screen.tsx (hunk 1 above)
Comment (jonahkagan) [reply apps/design/frontend/src/geography_screen.tsx]: Good idea 7fa4595c5

**Pull request merged**